modules/dkan/dkan_datastore/dkan_datastore.module

```diff
@@ -413,14 +414,12 @@ function dkan_datastore_file_link_field() {
 
 /**
  * Returns name of node type that the Datastore is attached to.
+ *
+ * DEPRECATED: Use Resource::resourceContentType
```

Comment:
Should we clean this up?

Reply:
@fmizzell this was more in case someone was using this in another module or something.